source/Launchpad-SUnit/LaunchpadTest.class.st

```diff
@@ -0,0 +1,86 @@
+Class {
+	#name : #LaunchpadTest,
+	#superclass : #TestCase,
+	#instVars : [
+		'memoryLogger',
+		'runningApplication'
+	],
+	#category : #'Launchpad-SUnit'
+}
+
+{ #category : #testing }
+LaunchpadTest class >> isAbstract [
+
+	^ self = LaunchpadTest
+]
+
+{ #category : #private }
+LaunchpadTest >> assertLogRecordsMatch: expectedLogEntries [
+
+	memoryLogger recordings with: expectedLogEntries do: [ :record :expectedLogEntry | 
+		self assert: ( record printString includesSubstring: expectedLogEntry ) ]
+]
+
+{ #category : #private }
+LaunchpadTest >> assertThereIsNoLogRecords [
+
+	self assertLogRecordsMatch: #(  )
+]
+
+{ #category : #running }
+LaunchpadTest >> runCase [
+
+	self shouldnt: [ super runCase ] raise: Exit
+]
+
+{ #category : #private }
+LaunchpadTest >> runMemoryLoggerDuring: aBlock [
```

Comment:
Wouldn't it be nice to have a `#runMemoryLoggerDuring:assertingLogsMatch:` message?